features/landrush-ip.feature

```diff
@@ -0,0 +1,63 @@
+Feature: landrush-ip
+  Landrush should pick the desired IP, given multiple network interfaces.
+
+  Scenario Outline: booting a box and picking a specific interface
+    Given a file named "Vagrantfile" with:
+    """
+    Vagrant.configure('2') do |config|
+      config.vm.box = '<box>'
+      config.vm.hostname = 'my-host.landrush-acceptance-test'
+
+      config.vm.network :private_network, ip: '192.168.50.50'
```

Comment:
Awesome, I really like that you use the Cucumber feature here. It would be great though, if the tests would use multiple interfaces via DHCP. Using static IPs the whole test becomes a bit implicit and for the static IP case the "easier" option is _config.landrush.host_ip_address_.  

If DHCP is used the test becomes also a bit more telling:

    Scenario Outline: booting a box and picking a specific interface
      Given a file named "Vagrantfile" with:
      """
      Vagrant.configure('2') do |config|
        config.vm.box = '<box>'
        config.vm.hostname = 'my-host.landrush-acceptance-test    
        config.vm.network :private_network, type: 'dhcp'
        config.vm.network :private_network, type: 'dhcp'
        config.vm.network :private_network, type: 'dhcp'
        config.vm.network :private_network, type: 'dhcp'
        config.vm.network :private_network, type: 'dhcp'    
        config.vm.synced_folder '.', '/vagrant', disabled: true    
        config.landrush.enabled = true
        config.landrush.tld = 'landrush-acceptance-test'
        config.landrush.interface = 'eth3'
      end
      """
      When I successfully run `bundle exec vagrant up --provider <provider>`
      Then the host visible IP address of the guest is the IP of interface eth3

      Examples:
        | box             | provider   |
        | debian/jessie64 | virtualbox 

One would then need a new step definition for the _then_ statement where one gets the IP address of eth3 via a _vagrant ssh_ command and makes sure that this is the host visible IP.

In this case the test becomes documentation itself. Might address some of @bexelbie concerns.